plugins/gradients/index.js

```diff
@@ -0,0 +1,12 @@
+module.exports = () => ({ addUtilities, theme, variants }) => {
```

Comment:
We just decided to use the new `plugin()` API that 1.2 introduced -- just means importing that fn and wrapping this export in it (can also skip the extra `() =>`). For now it doesnt do all that much -- mainly handles the invoking in `tailwind.config.js` automatically so no longer need to include `()` there -- but it allows some other features and feels like the "right" way to do it now

Example here: https://github.com/vigetlabs/tailwindcss-plugins/blob/master/plugins/rect/index.js